components/Modal/Modal.js

```diff
@@ -43,9 +47,16 @@ function Modal({
       isOpen={isOpen}
       onRequestClose={onRequestClose}
       shouldCloseOnOverlayClick={shouldCloseOnOverlayClick}
+      ariaHideApp={false}
```

Comment:
Why did this change? We definitely want to hide the entire app when a modal opens to trap focus for screen reader users.

Reply:
oh yeah I guess I have overlooked that part. It was due to an error I saw on console `App element is not defined`. I have addressed this issue following. Addressed this referencing this SO [article](https://stackoverflow.com/questions/48269381/warning-react-modal-app-element-is-not-defined-please-use-modal-setappeleme)

Reply:
Hmmm maybe some sort of regression, but we defined the app element a long time ago in [`pages/_app.js` (Line 106)](https://github.com/OperationCode/front-end/blob/main/pages/_app.js#L106)

Reply:
I'll make an issue for this. Please revert it 🙏

Reply:
yep

Reply:
actually - just gonna make a PR for this one